proto/regen/ecocredit/marketplace/v1/types.proto

```diff
@@ -5,9 +5,6 @@ package regen.ecocredit.marketplace.v1;
 import "gogoproto/gogo.proto";
 import "regen/ecocredit/marketplace/v1/state.proto";
 
-option go_package =
```

Comment:
We are not using `AllowDenomProposal` message. Removing this line is breaking `buf-lint` check.